test/e2e/_incluster-test-code/main_test.go

```diff
@@ -19,6 +19,10 @@ import (
 	"testing"
 
 	f "github.com/operator-framework/operator-sdk/pkg/test"
+
+	// This import tests double-registration of the --kubebuilder flag:
```

Comment:
@jmrodri @camilamacedo86 confirmed that this import causes a panic w/o these changes.